odo/backends/sql.py

```diff
@@ -172,7 +172,9 @@ def discover_typeengine(typ):
     if isinstance(typ, (sa.NUMERIC, sa.DECIMAL)):
         return datashape.Decimal(precision=typ.precision, scale=typ.scale)
     if isinstance(typ, (sa.String, sa.Unicode)):
-        return datashape.String(typ.length, typ.collation)
+        return datashape.String(typ.length, None)
```

Comment:
Vocabulary question here... by "encoding" do you mean (this)[https://docs.python.org/3/library/codecs.html#standard-encodings]? I can check.